diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed437c0342..4fd3d7dd33 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -273,7 +273,7 @@ endif()
 
 option(BUILD_WITH_WEBENGINE "BUILD_WITH_WEBENGINE" ON)
 if (BUILD_WITH_WEBENGINE)
-    find_package(Qt${QT_VERSION_MAJOR}WebEngineCore ${REQUIRED_QT_VERSION} CONFIG QUIET)
+    find_package(Qt${QT_VERSION_MAJOR}WebEngineCore ${REQUIRED_QT_VERSION} CONFIG)
     if(APPLE)
         set_package_properties(Qt${QT_VERSION_MAJOR}WebEngineCore PROPERTIES
             DESCRIPTION "Qt${QT_VERSION_MAJOR} WebEngineCore component."
